### PR TITLE
Added org.get_temba_client and temba_merge_contacts

### PR DESCRIPTION
**Pull request by rowanseymour**
Some functionality from ChatPro that will be shared by TracPro

### CODE REVIEW

File: dash/utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    """
+    Return the intersection of lists
+    """
+    if not args:
+        return []
+
+    result = set(args[0])
+    for l in args[1:]:
+        result &= set(l)
+
```
Comment (nicpottier) [inline dash/utils/__init__.py]: Think you can do something like:
```python
result = result.intersection(args[1:])
```

File: dash/utils/__init__.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    """
+    Return the union of lists
+    """
+    if not args:
+        return []
+
+    result = set(args[0])
+    for l in args[1:]:
+        result |= set(l)
+
```
Comment (nicpottier) [inline dash/utils/__init__.py]: same here.

**Comment by nicpottier** [comment]
Looks good.

File: dash/utils/__init__.py (hunk 1 above)
Comment (rowanseymour) [inline dash/utils/__init__.py]: It works with `list(set(args[0]).intersection(*args[1:]))`... more Pythonic?

**Pull request merged**